packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -386,17 +426,31 @@ export default class Meeting extends StatelessSparkPlugin {
   /**
    *
    * Leave the current meeting
+   * @param {Object} options An object of options
+   * @param {Boolean} options.keepLocalMedia Whether to tear down the local media connections or not.
    * @returns {Promise}
    * @memberof Meeting
    */
   leave() {
-    return this.meetingRequest.leaveMeeting({
-      locusUrl: this.locusUrl,
-      selfId: this.selfId,
-      correlationId: this.correlationId,
-      resourceId: null,
-      deviceUrl: this.deviceUrl
-    });
+    return this.meetingRequest
```

Comment:
we should send an event when the leave completes

Reply:
Ok. Will add